pkg/operations/start.go

```diff
@@ -177,9 +207,10 @@ func waitForSpawn(vm *api.VM) error {
 		time.Sleep(checkInterval)
 
 		if util.FileExists(path.Join(vm.ObjectPath(), constants.PROMETHEUS_SOCKET)) {
-			return nil
+			vmChans.SpawnFinished <- nil
+			return
 		}
 	}
 
-	return fmt.Errorf("timeout waiting for ignite-spawn startup")
```

Comment:
Please move https://github.com/weaveworks/ignite/pull/836/files#diff-fe685b5ad77268d55269c38f483185a644e6b741eddc7b86511bbe3d51fe0ad7L133-L136 here and call `providers.Client.VMs().Set(vm)` one more time.

```go
	// Set the start time for the VM
	startTime := apiruntime.Timestamp()
	vm.Status.StartTime = &startTime
	vm.Status.Network.Plugin = providers.NetworkPluginName
```

Goal is for waitForSpawn to update the VM object with the proper VM start time.